event-sourcing/src/main/test/java/com/iluwatar/event/sourcing/IntegrationTest.java

```diff
@@ -0,0 +1,115 @@
+/**
```

Comment:
The path of the test file is funny. It's usually `src/test/java`, right? At the moment `mvn clean install` doesn't run the test at all.